SmartPlaces.Facilities/lib/IngestionManager.Mapped/test/Data/site.json

```diff
@@ -0,0 +1,92 @@
+﻿{
+    "data": {
+        "sites": [
+            {
+                "description": null,
+                "exactType": "Site",
+                "id": "SITE44nUdjjbqSX1sEXuwEWucr",
+                "name": "Cure Corridor",
+                "dateCreated": "2023-04-05T06:29:52.356",
+                "dateUpdated": "2023-04-05T06:29:52.356",
+                "type": [
+                    "Class",
+                    "Entity",
+                    "Place",
+                    "Site"
+                ],
+                "buildings": [
+                    {
+                        "address": {
+                            "countryName": "United States",
+                            "dateCreated": "2023-04-25T05:59:46.467",
+                            "dateUpdated": "2023-04-25T05:59:46.467",
+                            "id": "METABCx712hSb9AGGvzJTtc3Ny",
+                            "locality": "Woodrow",
+                            "postalCode": "80757",
+                            "region": "Colorado",
+                            "streetAddress": "12188 Colorado Highway 71"
```

Comment:
Can you obfuscate some of this information?

Reply:
This is a test customer in the Mapped Sandbox. Not sure we need to worry about it.